src/index/IndexImpl.Text.cpp

```diff
@@ -1467,15 +1454,10 @@ size_t IndexImpl::getIndexOfBestSuitedElTerm(
 template <size_t I>
 void IndexImpl::getECListForWordsAndSingleSub(
     const string& words, const vector<array<Id, I>>& subres,
-    size_t subResMainCol, size_t limit, vector<array<Id, 3 + I>>& res) const {
+    size_t subResMainCol, size_t limit,
+    vector<array<Id, 3 + I>>& res) const {  // TODO: rewrite
```

Comment:
Is the `// TODO: rewrite` still active, or can this comment be removed, as you already rewrote this method?